language/move-command-line-common/src/parser.rs

```diff
@@ -445,6 +457,7 @@ mod tests {
             "0u64x",
             "0u6 4",
             "0u",
+            "_10",
```

Comment:
A few other negative tests might be useful here; e.g., `10_u8`, `5_bool`.

Reply:
done